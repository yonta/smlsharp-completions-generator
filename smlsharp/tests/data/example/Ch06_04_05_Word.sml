0w10;
0wxA;
