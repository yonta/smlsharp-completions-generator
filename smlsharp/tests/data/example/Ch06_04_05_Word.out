val it = 0wxa : word

val it = 0wxa : word
